.env.example

```
@@ -1,2 +1,3 @@
 NEXT_PUBLIC_BACKEND_URL=https://localhost:4000
 NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_test_YOUR_KEY
+NEXT_PUBLIC_SITE_URL=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SITE_URL key should go before the NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY key

(UnorderedKey)